e2e/jenkins/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -38,7 +38,8 @@ pipeline {
                     steps {
                         script {
                             sh """
-                              wget --no-verbose https://github.com/che-incubator/chectl/releases/latest/download/chectl-linux \\
+                               # wget --no-verbose https://github.com/che-incubator/chectl/releases/latest/download/chectl-linux
```

Comment:
Is it quick fix and you plan to rework it to use latest chectl? 
there is some archive for linux but I'm not sure how it's supposed to be used.
@benoitf you know more, looks like README.md contains outdated info

Reply:
It's quick fix.